src/App.js

```diff
@@ -1,62 +1,40 @@
-import React from 'react';
-import './App.css';
-import Navbar from "./components/Navbar"
+import React, { useState, useEffect } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import Typography from '@material-ui/core/Typography';
+import Container from '@material-ui/core/Container';
+import Grid from '@material-ui/core/Grid';
+import Paper from '@material-ui/core/Paper';
 import TextField from '@material-ui/core/TextField';
-import Button from '@material-ui/core/Button';
-import Select from '@material-ui/core/Select';
-import InputLabel from '@material-ui/core/InputLabel';
-import MenuItem from '@material-ui/core/MenuItem';
-import FormControl from '@material-ui/core/FormControl';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
 
+const useStyles = makeStyles(theme => ({
+  heroContent: {
+    backgroundColor: theme.palette.background.paper,
+    padding: theme.spacing(8, 0, 6),
+  },
+  heroButtons: {
+    marginTop: theme.spacing(4),
+  },
+  basicGrid: {
+    flexGrow: 1,
+  },
+  paper: {
+    padding: theme.spacing(2),
+    display: 'flex',
+    overflow: 'auto',
+    flexWrap: 'wrap',
+    flexDirection: 'column',
+    alignItems:"center",
+    justifyContent:"center",
+    minHeight:180
+  },
+}));
 
-class App extends React.Component {
-  constructor(props){
-    super(props);
-    this.timeout = 0
-    this.state = {
-      input: "",
-      output: "",
-      inputLanguage: "js",
-      outputLangauge: "py",
-    }
-    this.postRequest = this.postRequest.bind(this)
-    this.autoTranslate = this.autoTranslate.bind(this)
-  }
-  
-  postRequest() {
-    // TODO: add error checking before post and get request
-    const params = new URLSearchParams({
-      input: this.state.input,
-      in_lang: this.state.inputLanguage,
-      out_lang: this.state.outputLangauge
-    })
-
-    // submit user query to search bar
-    let query = this.state.input + " in "
-    if (this.state.outputLangauge === "js") {
-      query += "javascript";
-    }
-    if (this.state.outputLangauge === "py") {
-      query += "python";
-    }
-    this.querySearch(query);
-
-    fetch('https://cjsback.herokuapp.com/', {
-      method: 'POST',
-      headers: new Headers({
-        'Content-Type': 'application/x-www-form-urlencoded',
-      }),
-      body: params.toString()
-    })
-      .then(res => res.json())
-      .then((data) => {
-        this.setState({
-          output: data['response']
-        })
-      })
-  }
-
-  allowTabs() {
+function allowTabs() {
     const textbox = document.getElementById("input-text-area")
```

Comment:
This should probably use [refs](https://reactjs.org/docs/hooks-reference.html#useref) to be safer.

Reply:
added TODO